x/gov/keeper/constitution_test.go

```diff
@@ -0,0 +1,39 @@
+package keeper_test
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestApplyConstitutionAmendment(t *testing.T) {
+	govKeeper, _, _, ctx := setupGovKeeper(t)
+
+	tests := []struct {
+		name                string
+		initialConstitution string
+		amendment           string
+		expectedResult      string
+		expectError         bool
+	}{
+		{
+			name:                "failed patch application",
+			initialConstitution: "Hello World",
+			amendment:           "Hi World",
+			expectError:         true,
+		},
```

Comment:
no "success" test case ?

Reply:
Thanks, I actually left this file to readdress later and then forgot about it..

Reply:
done